src/main/java/de/oelrichsgarcia/studipTelegramBot/studipTelegramBot/studip/StudIPBot.java

```diff
@@ -257,12 +264,18 @@ public void downloadFilesAndCreateFolders(ArrayList<StudIPObject> objects, Path
                     if (object.getUpdated().getTime() > file.lastModified()) {
                         Sout("Updating outdated file " + basePath + "/" + object.getName() + " " + fileSize);
                         downloadFile((StudIPFile) object, basePath);
+                        if(useGoogleDrive) {
```

Comment:
Please use a custom Downloader implemnents Downloader

Reply:
Can't do that, cause it builds on top of the downloads. Files need to exist locally, before they can be uploaded,